packages/common/src/common.ts

```diff
@@ -962,7 +962,7 @@ export class Common {
   hardforks(): HardforkTransitionConfig[] {
     const hfs = this._chainParams.hardforks
     if (this._chainParams.customHardforks !== undefined) {
-      this._chainParams.customHardforks
+      return this._chainParams.customHardforks
```

Comment:
I'm not 100% sure what the goal of this code was, but it seems to be wrong without the `return`.